app/assets/stylesheets/psique/components/_fonticon.scss

```diff
@@ -0,0 +1,113 @@
+// *************************************
+//
+//   Custom Font
+//   -> Font implementation
+//
+// *************************************
+
+// -------------------------------------
+//   Font-embedded
+// -------------------------------------
+
+@font-face {
```

Comment:
As I mentioned below, this should go on vendors, and it should be generated with fontello gem.